tools/topology/topology2/cavs/CMakeLists.txt

```diff
@@ -24,9 +24,15 @@ PREPROCESS_PLUGINS=nhlt,NHLT_BIN=nhlt.bin"
 "cavs-passthrough-hdmi\;cavs-mixin-mixout-hda-4ch-mtl\;PLATFORM=mtl,\
 HDA_CONFIG=mix,NUM_DMICS=4,PDM1_MIC_A_ENABLE=1,PDM1_MIC_B_ENABLE=1,PREPROCESS_PLUGINS=nhlt,NHLT_BIN=nhlt-mtl-hda-mix.bin"
 
-# CAVS SDW topology with passthrough pipelines
+# SDW HEADSET
 "cavs-sdw\;cavs-sdw"
 
+# SDW HEADSET + AMPLIFIER
+"cavs-sdw\;cavs-sdw-1-amp\;NUM_SDW_AMPS=1"
+
+# SDW HEADSET + 2 AMPLIFIERs
+"cavs-sdw\;cavs-sdw-2-amp\;NUM_SDW_AMPS=2"
```

Comment:
I think you missed my feedback @bardliao 

We should start using topology names that can be directly used by the kernel, so something like rt711-l0-rt1316-l12-rt715-l3 is needed. It's not just about adding a variable number of amplifiers, we also need to deal with the NOJACK case. In other words all variations we had for topology1 need to be supported here.